OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -119,10 +153,19 @@ public override bool Tick(Actor self)
 
 			if (activeResupplyTypes == 0)
 			{
-				var aircraft = self.TraitOrDefault<Aircraft>();
 				if (aircraft != null)
 					aircraft.AllowYieldingReservation();
 
+				var actorHasNextActivity = self.CurrentActivity.NextActivity != null;
+				if (!actorHasNextActivity && repairableNear == null)
```

Comment:
TIL that `RepairableNear` was never hooked up to rally points. Can we please fix this while we are here?

Reply:
Personally, I found the move to rallypoint a bit annoying for RepairableNear, because
- the repaired actor isn't blocking the only repair slot if it doesn't move
- depending on where it was repaired, it may have to move around the entire building to reach the rallypoint
- as ship yards/sub pens are also production facilities, things can get crowded at the rp even faster if repaired actors move there as well

I don't really mind changing it anyway, just wanted to point this out.